test/glance_test.gleam

```diff
@@ -852,448 +852,190 @@ pub fn function_parameters_test() {
 }
 
 pub fn expression_int_test() {
-  "pub fn main() { 1 2 3 }"
-  |> glance.module()
+  "1 2 3"
```

Comment:
Could you put back the deleted tests please? 🙏

Reply:
None of the tests have been deleted. Now there is an API for parsing statements, not every statement needs to be tested in the context of a full module. I've just simplified the tests so that a test for int's is that the statement level and not a module level. The module parser uses the statement function. There is no loss of coverage with this change.